pkg/trivy/wrapper.go

```diff
@@ -1,13 +1,17 @@
 package trivy
 
 import (
+	"encoding/json"
 	"fmt"
 	"github.com/aquasecurity/harbor-scanner-trivy/pkg/etc"
 	"github.com/aquasecurity/harbor-scanner-trivy/pkg/ext"
 	log "github.com/sirupsen/logrus"
 	"golang.org/x/xerrors"
+	"io/ioutil"
 	"os/exec"
 	"strings"
+
+	ttypes "github.com/aquasecurity/trivy/pkg/types"
```

Comment:
I can add it. But I'm curious why is it done that way? If it's importable and useable as a library shouldn't we import it? I'm all for having less dependencies but in this case it feels like duplication when we can avoid it. 

I have removed it here: https://github.com/aquasecurity/harbor-scanner-trivy/pull/87/commits/92676f01afb2860bd2d182727e053e4c6726926c

Reply:
When we stared implementing the adapter service it wasn't that easy to import Trivy as lib as is today. So the only reason I'd like to stick to spawning trivy child process to get the version info is for the sake of consistency with processing scan requests.

If, for any reason, we'd like to use Trivy as a Go module instead of executable binary we should do it for both scans and version info.